dcrsqlite/apisource.go

```diff
@@ -889,12 +889,17 @@ func (db *wiredDB) GetExplorerTx(txid string) *explorer.TxInfo {
 	tx.Vin = inputs
 
 	outputs := make([]explorer.Vout, 0, len(txraw.Vout))
-	for _, vout := range txraw.Vout {
+	for i, vout := range txraw.Vout {
+		txout, err := db.client.GetTxOut(txhash, uint32(i), true)
+		if err != nil {
+			log.Warnf("Failed to determine if tx out is spent for ouput %d of tx %s", i, txid)
+		}
```

Comment:
for `if err == nil && txout == nil`, I'd say that we want to check with `getrawtransaction` if it is really just in mempool

I'm not really sure what the mempool arg for GetTxOut is supposed to help with

Reply:
nvm.  If it presently shows unconfirmed transaction outputs as unspent, then we're all good.